src/main/java/com/kmush12/CurrencyConverter/Converter/CurrencyConverterController.java

```diff
@@ -0,0 +1,33 @@
+package com.kmush12.CurrencyConverter.Converter;
+
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.math.BigDecimal;
+
+@RestController
+@RequestMapping("/convert")
+public class CurrencyConverterController {
+
+    private final CurrencyConverterService currencyConverterService;
+
+    @Autowired
+    public CurrencyConverterController(CurrencyConverterService currencyConverterService) {
+        this.currencyConverterService = currencyConverterService;
+    }
+
+    @GetMapping("{fromCode}/{toCode}/{amount}")
```

Comment:
Skoro przekazujesz tutaj aż 3 wartości to może lepiej i czytelniej będzie tutaj przekazywać obiekt JSONowy jako body requesta? Wtedy ścieżka byłaby `/conver` i jako JSON możesz podać te wymagane 3 pola.

Zrób tak w ramach ćwiczenia, będzie trochę czytelniej (chociaż tak jak teraz też jest jak najbardziej okej).